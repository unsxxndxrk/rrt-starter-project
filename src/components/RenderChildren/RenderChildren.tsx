import React from "react";

function RenderChildren({
  children,
}: any & {
  children?: React.ReactNode;
}) {
  return children;
}

export default RenderChildren;
