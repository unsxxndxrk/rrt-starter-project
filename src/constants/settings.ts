export const APP_DEFAULT_LANGUAGE: string = "en";
export const API_ENDPOINT: string =
  process.env.REACT_APP_API_ENDPOINT || window.location.origin;
