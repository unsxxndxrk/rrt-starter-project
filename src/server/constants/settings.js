const ORIGIN = "http://localhost";
const PORT = "8000";
const USERS_PATH = "./data/users.json";
const TODOS_PATH = "./data/todos.json";

exports.ORIGIN = ORIGIN;
exports.PORT = PORT;
exports.USERS_PATH = USERS_PATH;
exports.TODOS_PATH = TODOS_PATH;
