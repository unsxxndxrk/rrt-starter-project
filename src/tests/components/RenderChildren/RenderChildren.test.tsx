import { render } from "tests/settings/render";
import RenderChildren from "components/RenderChildren/RenderChildren";

const childrenMock: string = "children_mock";

describe("<RenderChildren />", () => {
  test("should render RenderChildren component", () => {
    const wrapper = render(<RenderChildren>{childrenMock}</RenderChildren>);
    expect(wrapper.asFragment()).toMatchSnapshot();
    expect(wrapper.getByText(childrenMock)).toBeTruthy();
  });
});
