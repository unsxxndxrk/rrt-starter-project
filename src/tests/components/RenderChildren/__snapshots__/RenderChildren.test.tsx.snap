// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<RenderChildren /> should render RenderChildren component 1`] = `
<DocumentFragment>
  children_mock
</DocumentFragment>
`;
